src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -9496,6 +9509,10 @@ interface AutocompleteData {
   txts: string[];
   enums: NSEnums;
   flags(schema: [string, string | number | boolean | string[]][]): { [key: string]: ScriptArg | string[] };
+  hostname: string;
+  filename: string;
```

Comment:
These need comments. (Really, they all do, but I'll only insist for new fields.)